docs/compilers/Visual Basic/Compiler Breaking Changes - post VS2019.md

```diff
@@ -1,27 +1,29 @@
 ## This document lists known breaking changes in Roslyn in *Visual Studio 2019 Update 1* and beyond compared to *Visual Studio 2019*.
 
-*Breaks are formatted with a monotonically increasing numbered list to allow them to referenced via shorthand (i.e., "known break #1").
-Each entry should include a short description of the break, followed by either a link to the issue describing the full details of the break or the full details of the break inline.*
+*Breaking changes are formatted with a numerically delineated list so as to allow shorthand numerical references (e.g., "known break #1").
+Each entry should include a short description of the breaking change, followed by either a link to the issue describing the full details of the change or the full details inline.*
 
```

Comment:
My main concern with rewording this is that every other breaking change document has this same wording. If we're going to update one, we need to update them all.

Reply:
I am confident I can replace these in existing files that I have access to, but what of new files? Is there a template that new files use to create the wording that would make it the same across all files that display this header (specifically "breaking changes" pages)?

Also, is there any process that is used to approve wording? I was somewhat concerned about this change myself wondering if the wording may have been used intentionally for some reason. If not, short of a "user test" for documentation (if such a thing exists) is there a process to finalize wording and esp. where templated headers and such are concerned?

Reply:
> I am confident I can replace these in existing files that I have access to, but what of new files?

We copy/paste from old files, so that is all that's needed.

> Also, is there any process that is used to approve wording?

This review process :slightly_smiling_face:.